core/trino-main/src/main/java/io/trino/security/AccessControlManager.java

```diff
@@ -214,6 +215,17 @@ public void addSystemAccessControl(SystemAccessControl systemAccessControl)
                         .build());
     }
 
+    @VisibleForTesting
```

Comment:
You don't need anything like this. Simply use `io.trino.testing.TestingAccessControlManager` which should be already used in query runner.

Reply:
systemAccessControls is private field and there is no access to it from upper class.

Reply:
If you use `TestingAccessControlManager` you will not be using `AcessControlManager` at all.  `TestingAccessControlManager` is already setup to do these kind of checks.  For examples of tests using `TestingAccessControlManager`, see `TestAccessControl` and `TestCallTask`

Reply:
Ok didn't notice it at begging. Thx for hint.